### PR TITLE
Make it clearer what is running in the github action UI

### PR DESCRIPTION
**Pull request by mattdangerw**

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -17,12 +17,12 @@ jobs:
       fail-fast: false
       matrix:
         backend: [tensorflow, jax, torch]
-        version: [stable]
+        version: [keras-stable]
         include:
           - backend: jax
-            version: 3.1
+            version: keras-3.1
```
Comment (SamanehSaadat) [inline .github/workflows/actions.yml:23]: Doesn't this need to be the latest Keras release?

Comment (mattdangerw) [reply .github/workflows/actions.yml:23]: This is a test we added recently to try running against Keras a few versions back. Just makes sure we are staying compatible.

Comment (SamanehSaadat) [reply .github/workflows/actions.yml:23]: Oh, that's great!

**Review by SamanehSaadat — APPROVED** [review]
Thanks, Matt!

**Pull request merged**